aten/src/ATen/native/native_functions.yaml

```diff
@@ -1863,7 +1863,11 @@
 - func: cudnn_batch_norm(Tensor input, Tensor weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float exponential_average_factor, float epsilon) -> (Tensor, Tensor, Tensor, Tensor)
   dispatch:
     CUDA: cudnn_batch_norm
-  autogen: cudnn_batch_norm.out
+
+# TODO: If possible, rename the out0 ... out3 to output, save_mean, save_var, reserve (bc break)
```

Comment:
nit: I don't think it's going to be worth BC-breaking to improve this internal name, so you can remove this todo.